src/server/game/Handlers/MovementHandler.cpp

```diff
@@ -26,12 +26,14 @@
 #include "MotionMaster.h"
 #include "MovementGenerator.h"
 #include "MoveSpline.h"
+#include "ObjectAccessor.h"
 #include "Transport.h"
 #include "Battleground.h"
 #include "InstanceSaveMgr.h"
 #include "ObjectMgr.h"
 #include "Vehicle.h"
 #include "GameTime.h"
+#include "GameClient.h"
```

Comment:
while you are at it you could order these alphabetically with worldsession on top 😝

Reply:
done